### DISCUSSION TITLE
Using meta-rust for including nativesdk- into SDK.

### DISCUSSION THREAD

**Discussion by ene-ilies** [meta-rust/meta-rust#351] [Q&A] [answered]
Hello,

My name is Bogdan Ilies and I am trying to use meta-rust for installing nativesdk- packages within my generated SDK.

Could you please help me with regard to what should I include into my local.conf in order to include that?

So far I have tried by including nativesdk-cargo like this:
`TOOLCHAIN_TARGET_TASK_append = " \ 
	nativesdk-cargo\
"`
and when running "bitbake core-image-full-cmdline -c populate_sdk" I am facing the following error: "Could not find specification for target "x86_64-pokysdk-linux". Use `--print target-list` for a list of built-in targets"
I have also tried to do it like this: 
`TOOLCHAIN_TARGET_TASK_append = " \ 
	packagegroup-rust-cross-canadian-${MACHINE} \
"`
but also no luck with it.
Am I trying to include the wrong packages or there might be a problem somewhere else.

I have to mention that I am quite new to yocto and I am sorry if this is a silly question.
Any help is very much appreciated.

Thank you,
Bogdan Ilies

**Comment by weltling** [comment, marked as answer]
This one should be the only needed:

`TOOLCHAIN_HOST_TASK_append = " packagegroup-rust-cross-canadian-${MACHINE}"`

On the other point - i only used `core-image-minimal` to build the SDK, `core-image-full-cmdline` could work but should confirm.

Also please be sure you use at least `dunfell`.

Thanks

**Reply by ene-ilies** [reply]
Hi @weltling 

Thank you very much for your answer.  I will try that right away.

Have a nice day,
Bogdan Ilies
